src/addButton.jsx

```diff
@@ -0,0 +1,5 @@
+import React from 'react';
+
+export default function AddButton({ text, onClick }) {
+  return (<button type="button" onClick={onClick}>{text}</button>);
+}
```

Comment:
```suggestion
export default function AddButton({ text, onClick }) {
  return (
    <button 
      type="button"
      onClick={onClick}
    >
        {text}
    </button>
  );
}
```

비교했을 때 어떠하신가요?

저는 한줄의 너무 많은 내용이 들어가면 오히려 가독성이 떨어지는 것 같아요.

Reply:
이게 훨씬 낫네요 ㅎㅎ